backon/src/backoff/constant.rs

```diff
@@ -90,6 +98,8 @@ impl BackoffBuilder for ConstantBuilder {
 
             attempts: 0,
             jitter: self.jitter,
+            #[cfg(not(feature = "std"))]
+            rng: fastrand::Rng::with_seed(self.seed),
```

Comment:
- Use `Rng::with_seed(self.seed)` is it's not `None`
- Otherwise, use `Rng::new()` here if `std` enabled.
- If neither set seed nor has `std`, use our default value. 

We can extra the value as a global constant, and make this behavior clear in `with_jitter_seed`